spec/collection-spec.coffee

```diff
@@ -1,18 +1,33 @@
-describe 'Brainstem.Collection', ->
-  collection = updateArray = null
+$ = require 'jquery'
+_ = require 'underscore'
+Backbone = require 'backbone'
```

Comment:
So do we need to make them global then in the JSON file if we're requiring them in most of our files?